lib/Test/WWW/Mechanize/Catalyst/WithContext.pm

```diff
@@ -10,6 +10,12 @@ extends 'Test::WWW::Mechanize::Catalyst';
 our $VERSION = "0.02";
 $VERSION = eval $VERSION;
 
+has ctx => (
+    is     => 'ro',
+    isa    => 'Catalyst',
+    writer => '_set_ctx',
+);
+
```

Comment:
An alternative might be to make this a lazy attribute that instantiates as the context for '/'. Otherwise the user will need to make a request before checking the context.